packages/core/src/repositories/commitsRepository/index.ts

```diff
@@ -38,7 +35,7 @@ function filterByStatusQuery({
   status,
 }: {
   status: CommitStatus
-  scope: typeof CommitsRepository.prototype.scope
+  scope: InferedReturnType<typeof buildCommitsScope>
```

Comment:
Infer return of a function. Very handy in this case so we can type scopes

Reply:
hmm I usually use ReturnType, not sure what's the diff